### PR TITLE
pre-Loader color Changed

### LINKED ISSUE #410: Change color of Pre-loader
The existing color is complete pink .Change color of Pre-loader to such which matches with theme of the project. 
**Screenshots**

### PR DESCRIPTION
**Pull request by AryanShinde**
#Change color of pre-loader

-changed color of pre-loader as per requirements

Closes: #410 

### Describe the changes you've made

Primarily changed the colour of preloader and fixed responsiveness of website and overflowing as well.

## Type of change
Front-end

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Mention any unusual behaviour of your code (Write NA if not)
NA

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by ankitapuri** [comment]
@7saikat7 please review it once and don't merge .

**Comment by AryanShinde** [comment]
@ankitapuri somethings wrong?

**Comment by ankitapuri** [comment]
> @ankitapuri somethings wrong?

Not until now , just need to review once.

**Comment by AryanShinde** [comment]
Ah okay!

File: static/css/preloader.css
Comment (ankitapuri) [inline static/css/preloader.css]: make the position fixed else if we scroll it will move.

Comment (AryanShinde) [reply static/css/preloader.css]: done!

**Review by ankitapuri — APPROVED** [review]

**Pull request merged**
